Header.h

```diff
@@ -75,36 +72,40 @@ class Planets2
 		{
 			days[i] = this->planets_list[i]->distance_from_sun;
 		}
-		qsort(days, n, sizeof(int), kamp1);
+
+
+		qsort(days, n, sizeof(int), vuraz);
+		
+
 		for (int i = 0; i < kd; i++)
 		{
-			std::cout << days[i] << " " << " ";
+		cout << days[i] << " " << " ";
 		}
 	}
 
-	void getinfo(std::string planet_n)
+	void get_info(std::string planet_n)
 	{
 		for (int i = 0; i < kd; i++) {
 			if (planet_n == planets_list[i]->name)
 			{
-				std::cout << this->planets_list[i]->name << " " << this->planets_list[i]->mass << "/" << "kg" << endl;
+			cout << this->planets_list[i]->name << " " << this->planets_list[i]->mass << "/" << "kg" << endl;
 			}
 		}
 	}
 };
 
 
 
-void Distance(Planet& first, Planet& second)
+void Distance(Planet& first, Planet& second)	
 {
-	int distan;
-	distan = first.distance_from_sun - second.distance_from_sun;
-	std::cout << distan << "-" << "riznuca vidstani" << endl;
+	int distance;
+	distance = first.distance_from_sun - second.distance_from_sun;
+	cout << distance << "\t" << "riznuca vidstani" << endl;
 }
 
 void Mass(Planet& first, Planet& second)
 {
-	int rmas;
-	rmas = first.mass - second.mass;
-	std::cout << rmas << "-" << "riznuca masu" << endl;
-}
\ No newline at end of file
+	int armas;
+	armas = first.mass - second.mass;
+	cout << armas << "\t" << "riznuca masu" << endl;
+}
```

Comment:
Header file is way too long, wpuld be better if you could divide it in smaller parts